src/pages/Home/index.jsx

```diff
@@ -1,5 +1,46 @@
+import useCommitStore from "../../feature/commit/store/useCommitStore";
+import messages from "../../shared/constants/messages";
+import getCommitList from "./api/getCommitList";
+
 const Home = () => {
-  return <h1>Init</h1>;
+  const setCommitList = useCommitStore((state) => state.setCommitList);
+
+  const handleRepoInfoSubmit = async (e) => {
+    e.preventDefault();
+
+    const formData = new FormData(e.target);
+    const formJson = Object.fromEntries(formData.entries());
+    const parsedURL = formJson.repositoryURL.split("/");
+    const githubIndex = parsedURL.indexOf("github.com");
+    const [organizationName, repositoryName] = parsedURL.slice(
+      githubIndex + 1,
+      githubIndex + 3
+    );
```

Comment:
이 로직은, url 전체에서 [organizationName, repositoryName] 를 뽑아내는 중요한 함수같아보여요!
pages/Home 폴더 이내, 혹은 shared/utils 안에 `extractGitInfoFromURL.js` 로 빼고, 

```suggestion

const extractGitInfoFromURL = () => {
    const formData = new FormData(e.target);
    const formJson = Object.fromEntries(formData.entries());
    const parsedURL = formJson.repositoryURL.split("/");
    const githubIndex = parsedURL.indexOf("github.com");
    const [organizationName, repositoryName] = parsedURL.slice(
      githubIndex + 1,
      githubIndex + 3
    );
    
    return {organizationName, repositoryName};
}
```

이런 형태로 유틸함수로 빼는게 어떨까요?

Reply:
확실히 이러한 로직은 분리해서 사용해도 좋겠네요! 반영하겠습니다~